elastic/kibana
# [ResponseOps][Alerts] Fix muted alerts query using wrong filter

## Summary

Reverts the `getMutedAlerts` filter to use rule ids instead of rule type ids that caused the muted alerts state to be always empty.

## To verify

1. Create a rule that fire alerts
2. Navigate to `Stack management > Alerts`
3. Click on any alert actions menu (•••)
4. Toggle on and off the alert muted state and check that the action updates accordingly and the slashed bell icon appears in the status cell when muted

## Release note

Fix alert mute/unmute action
